### PR TITLE
build: Update to .NET 9 SDK RC1

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: .github/workflows/uwp-autoconvert.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ concurrency:
   cancel-in-progress: true
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  UnoCheck_Version: '1.26.0-dev.6'
+  UnoCheck_Version: '1.26.0-dev.15'
```
Comment (Youssef1313) [inline .github/workflows/uwp-autoconvert.yml:16]:
```suggestion
  UnoCheck_Version: '1.26.0-dev.17'
```

File: build/ci/templates/dotnet-mobile-install-mac.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 parameters:
-  UnoCheck_Version: '1.26.0-dev.6'
+  UnoCheck_Version: '1.26.0-dev.15'
```
Comment (Youssef1313) [inline build/ci/templates/dotnet-mobile-install-mac.yml:2]:
```suggestion
  UnoCheck_Version: '1.26.0-dev.17'
```

File: build/ci/templates/dotnet-mobile-install-windows.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 parameters:
-  UnoCheck_Version: '1.26.0-dev.6'
+  UnoCheck_Version: '1.26.0-dev.15'
```
Comment (Youssef1313) [inline build/ci/templates/dotnet-mobile-install-windows.yml:2]:
```suggestion
  UnoCheck_Version: '1.26.0-dev.17'
```

File: .github/workflows/uwp-autoconvert.yml [hunk 4]
```diff
@@ -69,7 +69,7 @@ jobs:
         & dotnet tool update --global uno.check --version $env:UnoCheck_Version --add-source https://api.nuget.org/v3/index.json
       name: Install Uno Check
     - run: | 
-        & uno-check -v --ci --non-interactive --fix --skip androidsdk --skip androidemulator --skip xcode --skip gtk3 --skip vswin --skip vsmac --pre-major
+        & uno-check -v --ci --non-interactive --fix --skip androidemulator --skip xcode --skip gtk3 --skip vswin --skip vsmac --pre-major
```
Comment (Youssef1313) [inline .github/workflows/uwp-autoconvert.yml:72]:
```suggestion
        & uno-check -v --ci --non-interactive --fix --skip androidsdk --skip androidemulator --skip xcode --skip gtk3 --skip vswin --skip vsmac --pre-major
```

File: build/ci/templates/dotnet-mobile-install-linux.yml [hunk 5]
```diff
@@ -17,7 +17,6 @@ steps:
 
   - bash: |
       dotnet tool update --global uno.check --version ${{ parameters.UnoCheck_Version }} --add-source https://api.nuget.org/v3/index.json
-      # androidsdk disabled for https://github.com/unoplatform/uno.check/issues/241
-      uno-check --ci --non-interactive --fix --skip androidsdk --skip gtk3 --pre-major
+      uno-check --ci --non-interactive --fix --skip gtk3 --pre-major
```
Comment (Youssef1313) [inline build/ci/templates/dotnet-mobile-install-linux.yml:20]:
```suggestion
      # androidsdk disabled for https://github.com/unoplatform/uno.check/issues/241
      uno-check --ci --non-interactive --fix --skip androidsdk --skip gtk3 --pre-major
```

File: build/ci/templates/dotnet-mobile-install-mac.yml [hunk 6]
```diff
@@ -9,7 +9,7 @@ steps:
 
   - powershell: |
       & dotnet tool update --global uno.check --version ${{ parameters.UnoCheck_Version }} --add-source https://api.nuget.org/v3/index.json
-      & uno-check -v --ci --non-interactive --fix --skip androidsdk --skip androidemulator --skip xcode --skip gtk3 --skip vswin --skip vsmac --pre-major
+      & uno-check -v --ci --non-interactive --fix --skip androidemulator --skip xcode --skip gtk3 --skip vswin --skip vsmac --pre-major
```
Comment (Youssef1313) [inline build/ci/templates/dotnet-mobile-install-mac.yml:12]:
```suggestion
      & uno-check -v --ci --non-interactive --fix --skip androidsdk --skip androidemulator --skip xcode --skip gtk3 --skip vswin --skip vsmac --pre-major
```

File: build/ci/templates/dotnet-mobile-install-windows.yml [hunk 7]
```diff
@@ -9,7 +9,7 @@ steps:
 
   - powershell: |
       & dotnet tool update --global uno.check --version ${{ parameters.UnoCheck_Version }} --add-source https://api.nuget.org/v3/index.json
-      & uno-check -v --ci --non-interactive --fix --skip androidsdk --skip androidemulator --skip xcode --skip gtk3 --skip vswin --skip vsmac --pre-major
+      & uno-check -v --ci --non-interactive --skip androidemulator --skip xcode --skip gtk3 --skip vswin --skip vsmac --pre-major
```
Comment (Youssef1313) [inline build/ci/templates/dotnet-mobile-install-windows.yml:12]:
```suggestion
      & uno-check -v --ci --non-interactive --fix --skip androidsdk --skip androidemulator --skip xcode --skip gtk3 --skip vswin --skip vsmac --pre-major
```

**Comment by Youssef1313** [comment]
@jeromelaban Still failing

**Comment by Youssef1313** [comment]
docfx failure is a .NET runtime bug (https://github.com/dotnet/runtime/issues/105792). We have to wait for the next preview (if any) or RC1

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**